pkg/plugin/util/util.go

```diff
@@ -172,7 +172,17 @@ func ImplementWebhooks(filename string) error {
 	str, err = EnsureExistAndReplace(
 		str,
 		"// TODO(user): fill in your validation logic upon object update.",
-		`if r.Spec.Count < 0 {
+		`if oldRes.Spec.Count != 0 && newRes.Spec.Count < 0 {
+		return nil, errors.New(".spec.count must >= 0")
+	}`)
+	if err != nil {
+		return err
+	}
+
+	str, err = EnsureExistAndReplace(
+		str,
+		"// TODO(user): fill in your validation logic upon object deletion.",
+		`if res.Spec.Count < 0 {
```

Comment:
Here, we have extra lines.
Can we try to keep the change as clean as possible to make the review easier by keeping only the required changes?

Reply:
Reverted.